subsys/bluetooth/host/conn.c

```diff
@@ -992,8 +1009,12 @@ void bt_conn_set_state(struct bt_conn *conn, bt_conn_state_t state)
 			bt_conn_unref(conn);
 			break;
 		case BT_CONN_CONNECTED:
-		case BT_CONN_DISCONNECTING:
+			/* Can only happen if bt_disable is called whilst
+			 * in a connection.
+			 */
+			break;
 		case BT_CONN_DISCONNECTED:
+		case BT_CONN_DISCONNECTING:
```

Comment:
Please revert the order of `BT_CONN_DISCONNECTING and `BT_CONN_DISCONNECTED` so that it does not appear as a change